scripts/bundle_version.sh

```diff
@@ -1,8 +1,16 @@
+#!/usr/bin/env bash
+
 git=$(sh /etc/profile; which git)
-number_of_commits=$("$git" rev-list HEAD --count)
 
-bundle_version=$number_of_commits
+bundle_version=$1
+
+if [[ -z $bundle_version ]]; then
+  # If not provided as a command line argument, use the number of commits on the current branch
+  bundle_version=$("$git" rev-list HEAD --count)
+fi
 
+echo $bundle_version
+exit
```

Comment:
there isn't any usage for this script. i updated the usage for the execute.sh script though